lib/internal/util/inspect.js

```diff
@@ -63,6 +63,11 @@ const {
   isBigUint64Array
 } = require('internal/util/types');
 
+const assert = require('internal/assert');
+
+// Avoid monkey-patched built-ins.
+const { Object } = primordials;
```

Comment:
this won't prevent `delete Object.keys`. Try:
```suggestion
const { Object: { keys: objectKeys, getOwnPropertyNames: gOPNs } } = primordials;
```